src/pages/GovPages/Card.tsx

```diff
@@ -69,19 +70,27 @@ export const GovCard = ({ currency, currency1, apy, proposals }: CardProps) => {
   const theme = useContext(ThemeContext)
   const router = useRouter()
 
-  const onClick = () => {
+  const onCurrencyClick = () => {
```

Comment:
Can these 2 components be replaced with a `Link` in order to keep everything a little bit cleaner?